reo/process_results.py

```diff
@@ -234,6 +510,72 @@ def upfront_capex_after_incentives(self):
                                               (1 - self.inputs["Financial"]["offtaker_tax_pct"])
             return round(upfront_capex_after_incentives, 2)
 
+        def calculate_lcoe(self, tech_results_dict, tech_inputs_dict, financials):
+            """
+            The LCOE is calculated as annualized costs (capital and O+M translated to current value) divided by annualized energy
+            output
+            :param tech_results_dict: dict of model results (i.e. outputs from PVModel )
+            :param tech_inputs_dict: dict of model results (i.e. inputs to PVModel )
+            :param financials: financial model storing input financial parameters
+            :return: float, LCOE in US dollars per kWh
+            """
+            years = financials.analysis_years # length of financial life
+
+            if financials.two_party_ownership:
+                discount_pct = financials.owner_discount_pct
```

Comment:
You probably already sorted this with Dan, but just want to make sure: I believe that what is being calculated in the two party case here is the LCOE for the developer. In order to account for the developer's required rate of return (i.e. discount rate) I think that we should still discount items with the offtaker's rate but then multiply by the two party factor, which would calculate the offtaker's LCOE with the developer's "mark-up". Definitely a Dan question though (and `@dolis` doesn't seem to have a github.com equivalent).

Reply:
@NLaws Yes, this is the LCOE for the developer. I reviewed this with Dan and he said not to use the two party factor and to use the developer discount factor which accounts for their profit